File: Praktikum C/PRAK503-2210817210033-HariOctavianDelrossi.c
#include <stdio.h>
int maksimal(int a, int b){
    if(a > b){
        return a;}
    else{
        return b;}
}
int minimal(int a, int b){
    if(a < b){
        return a;}
    else{
        return b;}
}
int main(){
    int batas = 0;
    int maks = -100000; 
    int min = 100000; 
    int bilangan;
    scanf("%d", &bilangan); 
    while(batas < bilangan){
        int nilai; 
        scanf("%d", &nilai);
        maks = maksimal(maks, nilai); 
        min = minimal(min, nilai); 
        batas++;
    }   
    printf("%d %d",maks,min);
    return 0;
}

File: Praktikum Python/PRAK503-2210817210033-HariOctavianDelrossi.py
def maksimal (maks, nilai):
    if(maks > nilai): 
        return maks
    else: 
        return nilai

def minimal (min, nilai):
    if(min < nilai) : 
        return min
    else: 
        return nilai

batas = 0 
maks = -100000
min = 100000 
bilangan = int(input())
while batas < bilangan:
    nilai = map(int,input().split())
    for angka in nilai:
        maks = maksimal(maks, angka)
        min = minimal(min, angka)
        batas+=1
    print("%d %d"%(maks,min))